Information/CenterOfMass.FCMacro

```diff
@@ -97,6 +105,7 @@ class CenterofmassDock(QtGui.QDockWidget):
     def __init__(self):
         super().__init__()
         self.setAttribute(QtCore.Qt.WA_DeleteOnClose, True)    # free memory
+        self.setLocale(QtCore.QLocale.English)
```

Comment:
Can you please explain why you need to set the locale to English?

Reply:
The user interface texts are in English only. The setting ensures uniform decimal separator for line edit and spin box fields.